lib/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior.rb

```diff
@@ -30,27 +38,74 @@ def parent_collection_ids(valkyrie: false)
         return member_of_collection_ids if valkyrie
         member_of_collection_ids.map(&:id)
       end
+      # alias member_of_collection_ids child_object_ids # TODO: Cannot alias in this way because member_of_collection_ids method is already defined thru the attribute definition.
 
       ##
       # Gives the subset of #members that are PCDM objects
       # @return [Enumerable<ActiveFedora::Base> | Enumerable<Valkyrie::Resource>] an enumerable over the members
       #   that are PCDM objects
+      # NOTE: A collection could be added to members, but for Hyrax collections are stored using member_of_collection_ids.
+      #       As long as this rule holds, then members and ordered_members are equivalent to objects/child_objects.
+      # TODO: Do we want to support members separate from child_objects (see NOTE)
       def child_objects(valkyrie: false)
         af_objects = Wings::ActiveFedoraConverter.new(resource: self).convert.objects
         return af_objects unless valkyrie
         af_objects.map(&:valkyrie_resource)
       end
       alias objects child_objects
+      alias ordered_objects child_objects
```

Comment:
The #child_objects method returns the member objects in order, so #ordered_objects is aliased to #child_objects. This addresses part of Issue #3590.